rpcwebsocket.go

```diff
@@ -971,18 +971,18 @@ func (*wsNotificationManager) notifyNewTickets(clients map[chan struct{}]*wsClie
 	tnd *blockchain.TicketNotificationsData) {
 
 	// Create a ticket map to export as JSON.
-	var tickets []string
-	for _, h := range tnd.TicketsNew {
-		tickets = append(tickets, h.String())
+	ticketMap := make(map[string]string)
+	for i, ticket := range tnd.TicketsNew {
+		ticketMap[strconv.Itoa(i)] = ticket.String()
```

Comment:
This makes sense for the missed and spend notifications because the map is a hash key to a string value ("missed" or "spend"), but what's the point of a map here?  Or in the winning ticket notification for that matter?